### ISSUE TITLE
Add product to wish list lacks vocalized confirmation when product is added

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#15216]
**Describe the bug**
When the user adds a product to the wish list, there's no audible confirmation of this happening.

**Tell us the version of Spartacus**
- Library version: develop

<img width="1029" alt="Screen Shot 2022-02-22 at 10 35 53 AM" src="https://user-images.githubusercontent.com/15113219/155168792-b191cd26-44c9-45a1-9f19-8dfa6516e5da.png">

**Expected behaviour**
- Vocalized confirmation of item being added to wish list: _"This item has been added to your wish list"._

**Comment by KateChuen** [comment]
Tested on develop branch of spartacus using installation script
More extensive testing (on other screenreaders and devices) will be done with RC.
![wishlist](https://user-images.githubusercontent.com/45409515/158681127-efff8565-8219-44be-9287-b3810e12068d.png)
![Screen Shot 2022-03-16 at 4 04 01 PM](https://user-images.githubusercontent.com/45409515/158681144-def2b31f-b07e-459d-8efc-4c8edcf23ed3.png)
![Screen Shot 2022-03-16 at 4 04 15 PM](https://user-images.githubusercontent.com/45409515/158681152-5f8fe4c1-dfdb-43c6-a11e-0c3bc9c60fc4.png)
![Screen Shot 2022-03-16 at 4 04 32 PM](https://user-images.githubusercontent.com/45409515/158681163-7315e7e1-03c0-4f2b-9acf-7cdcba0c927c.png)
![Screen Shot 2022-03-16 at 4 04 40 PM](https://user-images.githubusercontent.com/45409515/158681179-142dd73f-5f19-42ba-9f9a-1544f910fb74.png)
